easytab.h

```diff
@@ -870,21 +879,41 @@ EasyTabResult EasyTab_Load_Ex(HWND Window,
 
 EasyTabResult EasyTab_HandleEvent(HWND Window, UINT Message, LPARAM LParam, WPARAM WParam)
 {
+    static bool LeftContext = true;
     PACKET Packet = { 0 };
-
     if (Message == WT_PACKET &&
         (HCTX)LParam == EasyTab->Context &&
         EasyTab->WTPacket(EasyTab->Context, (UINT)WParam, &Packet))
     {
         POINT Point = { 0 };
         Point.x = Packet.pkX;
         Point.y = Packet.pkY;
-        ScreenToClient(Window, &Point);
+        if(LeftContext && EasyTab->IsRelative)
+        {
+            Point.x = 0;
+            Point.y = 0;
+            LeftContext = false;
+        }
+        if(!EasyTab->IsRelative)
+        {
+            ScreenToClient(Window, &Point);
+        }
         EasyTab->PosX = Point.x;
         EasyTab->PosY = Point.y;
-
+        EasyTab->Buttons = Packet.pkButtons;
```

Comment:
I couldn't find it either but that's how I'm interpreting it and it seems to work